src/commands/down.ts

```diff
@@ -31,9 +41,40 @@ export default class Down extends Command {
       return false;
     }
 
+    let assuredDestroy = false;
+    if (flags.destroy) {
```

Comment:
I think that if they choose not to proceed the command should exit early instead of still running the down command without -v